shared/src/androidMain/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you refactor the main `shared` folder and project name to be `library` instead? I think this will make more sense when we're running gradle tasks to do releases later like `./gradlew :library:assembleRelease` instead of `./gradlew :shared:assembleRelease`. See this project as an example which uses `sdk` instead of `library`.